src/main/java/core/basesyntax/operation/BalanceOperation.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.operation;
+
+import core.basesyntax.db.FruitStorage;
+import core.basesyntax.model.FruitTransaction;
+
+public class BalanceOperation implements OperationHandler {
+    @Override
+    public void processOperation(FruitTransaction fruitTransaction) {
+        FruitStorage.fruitRepository.put(fruitTransaction.getFruit(),
+                fruitTransaction.getQuantity());
+    }

```

Comment:
According to the checklist, there should not be an empty line at the beginning of the class implementation. Please remove the empty line after the class declaration.